code/game/objects/explosion.dm

```diff
@@ -114,7 +114,8 @@ proc/explosion(turf/epicenter, devastation_range, heavy_impact_range, light_impa
 
 	return 1
 
-
+/mob/verb/camerashake(strength as num)

```

Comment:
This being a verb sounds like a mistake. A verb is something any player can initiate: players don't have a use for this.

Also, can't you find a better place to shove this into? `code/modules/mob/mob_helpers.dm` smells like a great place.